api/envoy/config/route/v3/route_components.proto

```diff
@@ -198,6 +198,13 @@ message VirtualHost {
   // If set and a route-specific limit is not set, the bytes actually buffered will be the minimum
   // value of this and the listener per_connection_buffer_limit_bytes.
   google.protobuf.UInt32Value per_request_buffer_limit_bytes = 18;
+
+  // Specify a set of default request mirroring policies for every routes under this virtual host.
+  // It takes precedence over the route config mirror policy entirely.
+  // That is, policies are not merged, most internal non-empty one becomes the mirror policies.
+  // Inherit from the request_mirror_policies of its route config only if the field of request_mirror_policies
```

Comment:
You can drop all these "Inherit from.." sentences. I think they are redundant as you clearly explain the precedence relationship in the preceding sentences.